### PR TITLE
[docs] Release notes for 2024.1.3.0

### PR DESCRIPTION
**Pull request by aishwarya24**
@netlify /preview/releases/ybdb-releases/v2024.1/

### CODE REVIEW

File: docs/content/preview/releases/ybdb-releases/v2024.1.md [hunk 1]
```diff
@@ -33,6 +33,145 @@ In future releases, the memory division settings will be used to determine how m
...
+
+#### YSQL
+
+* Introduces the `yb_make_next_ddl_statement_nonincrementing` configuration parameter to reduce catalog version increments during many consecutive DDLs, lowering memory and latency spikes. {{<issue 23786>}}
+* Added yb_make_next_ddl_statement_nonincrementing to YbDbAdminVariables for yb_db_admin role. {{<issue 23786>}}
+* Updates advisory lock error message with a workaround hint instead of `not yet implemented.` {{<issue 23881>}}
+
+#### YCQL
+
+* Updates VCQL tables `system_schema.functions` and `system_schema.aggregates` to align with Cassandra, aiding Cortex migration. {{<issue 3893>}}
```
Comment (ddhodge) [inline docs/content/preview/releases/ybdb-releases/v2024.1.md:94]: VCQL?

File: docs/content/preview/releases/yba-releases/v2024.1.md [hunk 2]
```diff
@@ -27,6 +27,73 @@ If you have a Replicated installation, you must migrate from Replicated to YBA I
...
+   <a href="https://downloads.yugabyte.com/releases/2024.1.3.0/yba_installer_full-2024.1.3.0-b105-linux-x86_64.tar.gz">
+     <i class="fa-brands fa-linux"></i>
+     <span>Linux x86</span>
+   </a>
+ </li>
+</ul>
+
+### New features
+
+* [Validation of Certificate Authority settings](/stable/yugabyte-platform/security/authentication/oidc-authentication-aad/). When configuring an external Certificate Authority to manage SSL certs used in Encryption in Transit (EIT), YugabyteDB Anywhere now validates the settings up front, thereby improving usability.
```
Comment (ddhodge) [inline docs/content/preview/releases/yba-releases/v2024.1.md:49]:
```suggestion
* [Validation of Certificate Authority settings](/stable/yugabyte-platform/security/authentication/oidc-authentication-aad/). When configuring an external Certificate Authority to manage SSL certificates used for encryption in transit, YugabyteDB Anywhere now validates the settings up front, improving usability.
```

Comment (ddhodge) [reply docs/content/preview/releases/yba-releases/v2024.1.md:49]: Link seems unrelated. Maybe stable/yugabyte-platform/create-deployments/create-universe-multi-zone/#encryption-settings?

Comment (aishwarya24) [reply docs/content/preview/releases/yba-releases/v2024.1.md:49]: You’re right.. its an incorrect IDEA. Changing it. Thanks.

**Pull request merged**